apps/AEPSampleAppNewArchEnabled/app/TargetView.tsx

```diff
@@ -0,0 +1,213 @@
+/*
+Copyright 2024 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import React from 'react';
+import {Button, ScrollView, StyleSheet, Text, View} from 'react-native';
+import {
+  Target,
+  TargetOrder,
+  TargetParameters,
+  TargetPrefetchObject,
+  TargetProduct,
+  TargetRequestObject,
+} from '@adobe/react-native-aeptarget';
+import {  useRouter } from 'expo-router';
+
+function TargetView() {
+  const targetExtensionVersion = async () => {
+    const version = await Target.extensionVersion();
+    console.log(`AdobeExperienceSDK: Target version: ${version}`);
+  };
+
+  const clearPrefetchCache = () => Target.clearPrefetchCache();
+
+  const getSessionId = async () => {
+    const sessionId = await Target.getSessionId();
+    console.log(`Session ID: ${sessionId}`);
+  };
+
+  const getThirdPartyId = async () => {
+    const id = await Target.getThirdPartyId();
+    console.log(`AdobeExperienceSDK: Third Party ID: ${id}`);
+  };
+
+  const getTntId = async () => {
+    const id = await Target.getTntId();
+    console.log(`AdobeExperienceSDK: TNT ID ${id}`);
+  };
+
+  const resetExperience = () => Target.resetExperience();
+
+  const setPreviewRestartDeeplink = () =>
+    Target.setPreviewRestartDeeplink('https://www.adobe.com');
+
+  const setSessionId = () => Target.setSessionId('sessionId');
+
+  const setTntId = () => Target.setTntId('tntId');
+
+  const setThirdPartyId = () => Target.setThirdPartyId('thirdPartyId');
+
+  const retrieveLocationContent = () => {
+    const mboxParameters1 = {status: 'platinum'};
+    const mboxParameters2 = {userType: 'Paid'};
+    const purchaseIDs = ['34', '125'];
+
+    const targetOrder = new TargetOrder('ADCKKIM', 344.3, purchaseIDs);
+    const targetProduct = new TargetProduct('24D3412', 'Books');
+    const parameters1 = new TargetParameters(mboxParameters1);
+    const request1 = new TargetRequestObject(
+      'sdk_smoke_tests_target',
+      parameters1,
+      'defaultContent1',
+      (error, content) => {
+        if (error) {
+          console.error(error);
+        } else {
+          console.log('Adobe content:' + content);
+        }
+      },
+    );
+
+    const parameters2 = new TargetParameters(
+      mboxParameters2,
+      {profileParameters: 'parameterValue'},
+      targetProduct,
+      targetOrder,
+    );
+    const request2 = new TargetRequestObject(
```

Comment:
Same comment here with `request1` and `request2`. In Android and iOS they don't pass in any TargetParameters. Totally optional to keep it in sync but might be easier to validate when smoke testing

Reply:
code updated